ml/src/main/spark_2.2/org/apache/spark/ml/h2o/SparkSpecificUtils.scala

```diff
@@ -0,0 +1,57 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package org.apache.spark.ml.h2o
+
+import org.apache.spark.SparkContext
+import org.apache.spark.ml.h2o.models.CrossSparkUtils
+import org.apache.spark.ml.param.Params
+import org.apache.spark.ml.util.DefaultParamsReader
+import org.apache.spark.ml.util.DefaultParamsReader.Metadata
+import org.json4s.jackson.JsonMethods._
+import org.json4s.{DefaultFormats, JObject}
+
+object SparkSpecificUtils extends CrossSparkUtils {
+
+  private def getAndSetParams(
+                               instance: Params,
```

Comment:
Is it formatted with scalafmt?

Reply:
No, formatting with Idea default configuration -> I think that we agreed to format using the spark way after we create the configuration file and put it in a repo or did I miss anything?

Reply:
Spark use scalafmt, they use a custom configuration of it to be mere restrictive (number chars on lines etc.) I think we can use scalafmt defaults for now.

Reply:
@mn-mikke oki, please create PR and let's start applying after that PR is in master

Reply:
:-) yep, no problem

Reply:
Thanks! I'm all for consistent code, but it will be easier to enforce once we have the template in the code https://0xdata.atlassian.net/browse/SW-1391